### DISCUSSION TITLE
读懂源代码的准备工作

### DISCUSSION THREAD

**Discussion by ttkltll** [yihong0618/running_page#469] [Q&A] [answered]
yihong,你好。 我是听《开源面对面》知道这个项目的。先做个自我介绍：我跑步也很多年，11年就开始跑步吧。当时健身，教练说健身要有氧运动辅助。后来反客为主， 跑步成主要，健身反成次要。也行性格使然，我更喜欢跑步，享受跑步之后那种宁静通透的感觉。所以你这个项目我很感兴趣。
我算是初次接触开源。虽然github帐号注册多年，但一直都是作为一个旁观者 一个使用者的角色。很少深入参与到开源社区里面去做些输出。我目前想把你这个项目的源代码读一读，然后把issues里面的一些问题给解决。你能提供一些资料给我吗？需要重点看哪些资料？比如我对什么gpx,tcx,fit文件就比较陌生。
目前想先解决这三个Issues: 
标签里的带good first issue标签的两个
https://github.com/yihong0618/running_page/issues/430

**Comment by yihong0618** [comment, marked as answer]
@ttkltll 不太需要读源码，其实看懂 README 那张图就行。可以从跑起来这个项目做起。
前端的话我们可能未来会改成纯 React 可以看看这方面的资料。

**Reply by ttkltll** [reply]
好的，谢谢回复。
